src/styles/_global.scss

```diff
@@ -10,3 +10,14 @@
     grid-template-columns: repeat(8, 1fr);
   }
 }
+
+.overlay-is-active {
+  height: 100%;
+  background-color: color('gray-darkest');
+  overflow: hidden;
+}
+
+.overlay-is-inactive {
+  background-color: color('white');
+  overflow: auto;
+}
```

Comment:
This should be the default state of the document. It shouldn't be a class. `overlay-is-active` should just override it!

Also, the above function isn't being called anywhere, so this class is never used in this code